### PR TITLE
[close #303] add a integration test case for disk full

### LINKED ISSUE #303: cdc: cdc need a test case for disk full
## Feature Request

**Is your feature request related to a problem? Please describe:**

cdc need a test case for disk full

**Describe the feature you'd like:**

add a test case for cdc when server disk is full
 
**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by zeminzhou**
Signed-off-by: zeminzhou <zhouzemin@pingcap.com>

### What problem does this PR solve?

Issue Number: close #303

Problem Description: **TBD**

### What is changed and how does it work?
Add a test case for integration test to test cdc when server disk is full.

### Code changes

- No code

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)

### Side effects

- No side effects

### Related changes

- No related changes

### CODE REVIEW

File: cdc/tests/integration_tests/disk_full/run.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	esac
+
+	tikv-cdc cli changefeed create --start-ts=$start_ts --sink-uri="$SINK_URI" --changefeed-id=$CF_ID
+
+	rawkv_op $UP_PD put 5000
+
+	sleep 3
+	state=$(tikv-cdc cli changefeed list --pd=$UP_PD | jq .[0]."summary" | jq ."state" | tr -d '"')
+	if [[ "$state" == "normal" ]]; then
+		exit 1
```
Comment (pingyu) [inline cdc/tests/integration_tests/disk_full/run.sh:40]: Suggest to print error for easy to debug.

File: cdc/tests/integration_tests/disk_full/run.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	fi
+
+	pid=$(ps -aux | grep "tikv-cdc" | awk '{print $2}' | head -n1)
```
Comment (pingyu) [inline cdc/tests/integration_tests/disk_full/run.sh:43]: Suggest to use `pgrep -f "tikv-cdc"`. `ps aux | grep "tikv-cdc"` will have a change to get the pid of `grep`.

File: cdc/tests/integration_tests/disk_full/run.sh [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+	check_sync_diff $WORK_DIR $UP_PD $DOWN_PD
+	rawkv_op $UP_PD delete 5000
+	check_sync_diff $WORK_DIR $UP_PD $DOWN_PD
+
+	cleanup_process $CDC_BINARY
+}
+
+trap stop_tidb_cluster EXIT
+run $*
+check_logs $WORK_DIR
```
Comment (pingyu) [inline cdc/tests/integration_tests/disk_full/run.sh:60]: Suggest to `check_logs_contains` to verify that the failpoint is effective.

File: cdc/tests/integration_tests/disk_full/run.sh (hunk 1 above)
Comment (zeminzhou) [reply cdc/tests/integration_tests/disk_full/run.sh:40]: Added, thanks！

File: cdc/tests/integration_tests/disk_full/run.sh (hunk 2 above)
Comment (zeminzhou) [reply cdc/tests/integration_tests/disk_full/run.sh:43]: Good suggestion, get it! Thank!

File: cdc/tests/integration_tests/disk_full/run.sh (hunk 3 above)
Comment (zeminzhou) [reply cdc/tests/integration_tests/disk_full/run.sh:60]: I don't think it's necessary. Becuase if failpoint isn't effective, the changefeed will be `normal`

File: cdc/tests/integration_tests/disk_full/run.sh [hunk 4]
```diff
@@ -37,10 +37,11 @@ EOF
 	sleep 3
 	state=$(tikv-cdc cli changefeed list --pd=$UP_PD | jq .[0]."summary" | jq ."state" | tr -d '"')
 	if [[ "$state" == "normal" ]]; then
+		echo "expected state is normal, but get $state"
```
Comment (pingyu) [inline cdc/tests/integration_tests/disk_full/run.sh:40]:
```suggestion
		echo "state is NOT expected to be normal"
```

**Review by pingyu — APPROVED** [review]
LGTM~

**Review by haojinming — APPROVED** [review]
LGTM

**Pull request merged**